### PR TITLE
[11.0] [MIG] account_bank_statement_import_mt940_base

### PR DESCRIPTION
**Pull request by feketemihai**
Work is continued from https://github.com/OCA/bank-statement-import/pull/125, migration to 10.0, not merged yet.

### CODE REVIEW

**Comment by feketemihai** [comment]
Runbot is falling with 'No space left on the device'

**Comment by melroy89** [comment]
Nice.. But nobody merges the PR anymore?

**Comment by pedrobaeza** [comment]
Again no if nobody contributes with their reviews

**Comment by melroy89** [comment]
@pedrobaeza Ah, makes sense. Well, maybe we should create a CONTRIBUTING.md file? Which explains it all. https://help.github.com/articles/setting-guidelines-for-repository-contributors/

**Comment by pedrobaeza** [comment]
You'll already have it in https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md, but we can't maintain one for each OCA repository.

File: account_bank_statement_import_mt940_base/tests/test_import_bank_statement.py
Comment (melroy89) [inline account_bank_statement_import_mt940_base/tests/test_import_bank_statement.py:1]: @feketemihai Could you also test the base module for SNS Bank MT940 files (create an extra test), just to be sure besides Rabobank & ING. See attachment for an example file. 

[tr-info_15785279_20180104123832.zip](https://github.com/OCA/bank-statement-import/files/1603485/tr-info_15785279_20180104123832.zip)

And even documentation (Dutch too bad) if needed:
[6_0016_35_1511_Informatiewijzer_MT940_rapportage.pdf](https://github.com/OCA/bank-statement-import/files/1603487/6_0016_35_1511_Informatiewijzer_MT940_rapportage.pdf)

If the test fails on some SNS specific thing, let me know I can help translating to English if needed.

**Review by melroy89 — CHANGES_REQUESTED** [review]
Rework needed, by adding an extra testcase for SNS Bank, besides ING & Rabobank.

File: account_bank_statement_import_mt940_base/tests/test_import_bank_statement.py
Comment (melroy89) [reply account_bank_statement_import_mt940_base/tests/test_import_bank_statement.py:1]: Ps. look also at the V10 PR: https://github.com/OCA/bank-statement-import/pull/125

Where I also propose to add a SNS MT940 testcase, maybe we should re-invent the wheel if not needed.

Comment (feketemihai) [reply account_bank_statement_import_mt940_base/tests/test_import_bank_statement.py:1]: @danger89 I've added SNS testcase, the ideea is that the module doesn't handle multiple statements in the same file, so i update the file by reducing the number of statements to the first one, so to import it you will have to export daily statements, not from a period of time...

Comment (melroy89) [reply account_bank_statement_import_mt940_base/tests/test_import_bank_statement.py:1]: @feketemihai 
But the modules based on this mt940_base module like `mt940_nl_ing` and `mt940_nl_rabo` (and soon `mt940_nl_sns`) will hopefully add support for importing statements from a period of time??

Since its unacceptable to import back account statements day by day..

Comment (feketemihai) [reply account_bank_statement_import_mt940_base/tests/test_import_bank_statement.py:1]: @danger89 i will try to add it in base at the begining of next week.

**Review by melroy89 — APPROVED** [review]

**Comment by feketemihai** [comment]
@astirpe Can i have your review here, please...

**Review by astirpe — APPROVED** [review]
Code review: OK.

**Comment by feketemihai** [comment]
@astirpe Thanks for review and quick reaction.

**Comment by alexriemer** [comment]
Hi guys. Is there anything missing except the merge? I could check the functionality on a separate system with test data, but need some help to get the code.

**Comment by feketemihai** [comment]
@pedrobaeza Can i have you review, please, it already have 2 approvals from some time...Thanks.

**Comment by pedrobaeza** [comment]
Can you squash a bit the commit history?

**Comment by feketemihai** [comment]
@pedrobaeza Ready...

**Comment by feketemihai** [comment]
@pedrobaeza Thanks for quick reaction.

**Pull request merged**
